### DISCUSSION TITLE
Q:

### DISCUSSION THREAD

**Discussion by xiaohao8** [walinejs/waline#1951] [Q&A] [answered]
### Describe the bug | 问题描述

![Screenshot_20230426-233456](http://img.202271.xyz:57549/i/2023/04/26/12o8lik.png)
配置了，无法发邮件，请问是yes填错了，还是因为yandex.com和yandex.ru不一样？
https://202271.xyz

### Website URL | 问题网站

cm.202271.xyz

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

**Comment by xiaohao8** [comment]
smtp密码用的是yandex生成的密钥

**Comment by lizheming** [comment, marked as answer]
去 Vercel 后台看 Log 有 node mailer 详细的错误信息。如果你想，你可以自己本地导入 node-mailer 并使用你的账号密码进行调试。

**Reply by xiaohao8** [reply]
![Screenshot_20230427-225434](http://img.202271.xyz:57549/i/2023/04/27/11amzxi.png)

**Comment by xiaohao8** [comment]
日志显示如图
